timefold-solver-python-core/src/main/python/score/_score_analysis.py

```diff
@@ -339,8 +340,8 @@ def constraint_match_total_map(self) -> dict[str, ConstraintMatchTotal]:
                                              constraint_name=e.getValue().getConstraintRef().constraintName()),
                 constraint_match_count=e.getValue().getConstraintMatchCount(),
                 constraint_match_set=_map_constraint_match_set(e.getValue().getConstraintMatchSet()),
-                constraint_weight=e.getValue().getConstraintWeight(),
-                score=e.getValue().getScore()
+                constraint_weight=to_python_score(e.getValue().getConstraintWeight()),
```

Comment:
Is this correct?

Reply:
Yes; constraintWeight is a core (see penalizeConfigurable)